providers/google.go

```diff
@@ -62,4 +64,33 @@ func init() {
 			Write:     false,
 		},
 	})
+
+	// GoogleAds Support Configuration
+	SetInfo(GoogleAds, ProviderInfo{
+		DisplayName: "Google Ads",
+		AuthType:    Oauth2,
```

Comment:
@sanjay-a01 please look at https://github.com/amp-labs/connectors/pull/666 and add the same params here too

Reply:
@RajatPawar I will look into it.